File: Hw_1/InversionProblem.py
_ = input()
A = list(map(int, input().split()))
B = list(map(int, input().split()))
C = list(map(int, input().split()))

cnt = 0

for a in A:
    for b in B:
        if a < b:
            continue
        for c in C:
            if b < c:
                cnt += 1
print(cnt)

File: Hw_1/InversionProblem.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int na, nb, nc, input, cnt = 0;
    vector<int> A, B, C;
    cin >> na >> nb >> nc;
    for (int i = 0; i < na; i++) {
        cin >> input;
        A.push_back(input);
    }
    for (int i = 0; i < nb; i++) {
        cin >> input;
        B.push_back(input);
    }
    for (int i = 0; i < nc; i++) {
        cin >> input;
        C.push_back(input);
    }

    for (int i = 0; i < na; i++) {
        for (int j = 0; j < nb; j++) {
            if (A[i] > B[j]) {
                for (int k = 0; k < nc; k++)
                    if (B[j] < C[k])
                        cnt++;
            }
        }
    }
        
    cout << cnt;

    return 0;
}